src/Gir/Generation/IGeneratableExtensions.cs

```diff
@@ -50,6 +50,16 @@ static void GenerateImport (this ICallable callable, IGeneratable parent, Indent
 			var retType = GetReturnCSharpType (callable, writer);
 
 			var (typesAndNames, names) = BuildParameters (callable, writer.Options, appendInstanceParameters: true);
+
+			// TODO: Better than using the constant string, insert a custom generatable which contains the import string as a constant.
+			// Also, glib-2.0.gir is problematic, it contains 2 library names :(
```

Comment:
Usually everything is in the first specified library

Reply:
I think I even talked about that issue with the gi devs and they said the same.

Reply:
Cool, we can do that.